### ISSUE TITLE
TSS map legend and description text

### ISSUE THREAD

**Issue by ericka-hegeman** [stormwaterheatmap/website#41]
1. On the map, the legend has an odd range of values (0. 7.07, and 50.0) mg/L. In the description, everything is in micrograms/L, not milligrams, so there's a mismatch there. **Note that this issue is present on other data layers (Runoff, for example).
2. In the palette table, all raster values are NA ,which makes this table not particularly meaningful. The equation has a * before paved, but not before traffic or rainfall. It seems like they should all be identical:
ln[TSS]=10.17 + 0.14(cumulative daily rainfall) + 0.21(traffic) + 0.28*(paved)
3. There is text talking about the parameter delta model but no model is shown after the colon.
4. The first sentence of the last paragraph is very long and hard to follow.

**Comment by cnilsen** [comment]
Will update docs with a general pollutant load/concentration description. Will direct to the technical methods. Placeholder in docs for now.
